package.json

```diff
@@ -21,10 +21,11 @@
   },
   "dependencies": {
     "@hmcts/nodejs-logging": "^3.0.0",
-    "@types/dotenv": "^6.1.0",
+    "@types/lodash": "^4.14.121",
     "request": "^2.83.0",
     "request-promise-native": "^1.0.5",
-    "yarn": "^1.13.0"
+    "yarn": "^1.13.0",
+    "lodash": "^4.17.11"
```

Comment:
can you just import the function you need, i.e. https://www.npmjs.com/package/lodash.merge

Reply:
sure